.github/workflows/dev.yaml

```diff
@@ -50,6 +50,9 @@ jobs:
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
+      - name: Get IBC Tokens
+        run: yarn --cwd packages/sdk-ui-ts get-ibc-tokens || true
```

Comment:
the `|| true` will make the build continue even if error occurs when running `get-ibc-tokens` script